experiments/build-swiftshader.md

```diff
@@ -0,0 +1,37 @@
+# Building SwiftShader with Coverage Enabled
+
+The SwiftShader CMakeLists.txt file defines the SWIFTSHADER\_EMIT\_COVERAGE flag to generate coverage information. However, it was designed for a very specific toolchain (clang-10 toolchain, CMake option SWIFTSHADER\_LLVM\_VERSION=10.0).
+
+The build at https://swiftshader-review.googlesource.com/c/SwiftShader/+/44728 can be built with coverage information without these restrictions.
+
+## Steps:
+
+1. Clone the [repo](https://swiftshader-review.googlesource.com/c/SwiftShader/+/44728)
```

Comment:
I suggest removing all text between "Building SwiftShader with Coverage Enabled" and "Steps:". It is no longer true. `SWIFTSHADER_EMIT_COVERAGE` always works now.

I suggest changing "Clone the ..." to:

`git clone https://github.com/google/swiftshader.git 9e718f962f87c30d08e91053f0e9ce3467cbd488`

I just picked the latest commit (9e718f). You do not need to clone CL 44728.